external_search.py

```diff
@@ -2354,7 +2354,7 @@ def audiences(self):
         # At this point we know we have a specific list of audiences.
         # We're either going to return that list as-is, or we'll
         # return that list plus ALL_AGES.
-        with_all_ages = as_is + [Classifier.AUDIENCE_ALL_AGES]
+        with_all_ages = list(as_is) + [Classifier.AUDIENCE_ALL_AGES]
```

Comment:
This is an unrelated change I saw while testing the NoveList integration. I don't know where this happens, but at some point `Filter.audiences` is set to a `set`, and appending a list to it causes an error. But there's no reason why a `set` shouldn't be allowed here.